src/main/java/alfio/controller/api/v2/user/ReservationApiV2Controller.java

```diff
@@ -709,6 +737,90 @@ public ResponseEntity<Boolean> removeCode(@PathVariable String reservationId, @R
         return ResponseEntity.ok(res);
     }
 
+    @GetMapping("/reservation/{reservationId}/get-applicable-custom-payment-method-details")
+    public ResponseEntity<List<UserDefinedOfflinePaymentMethod>> getApplicableCustomPaymentMethodDetails(@PathVariable String reservationId, Principal principal) throws PassedIdDoesNotExistException {
+        var reservation = ticketReservationManager
+            .findById(reservationId)
+            .orElseThrow(() -> new PassedIdDoesNotExistException("Passed reservationId does not exist."));
+
+        validateAccessToReservation(principal, reservation);
+
+        var event = eventRepository.findByReservationId(reservationId);
+        var allowedPaymentMethods = customOfflineConfigurationManager.getAllowedCustomOfflinePaymentMethodsForEvent(event);
+
+        allowedPaymentMethods
+            .forEach(paymentMethod ->
+            paymentMethod
+                .getLocalizations()
+                .forEach((key, locale) -> {
+                    Map<String, String> localeTexts = new HashMap<>();
+                    localeTexts.put("instructions", locale.getPaymentInstructions());
+                    localeTexts.put("description", locale.getPaymentDescription());
+                    localeTexts = Formatters.applyCommonMark(localeTexts);
+
+                    locale.setPaymentInstructions(localeTexts.get("instructions"));
+                    locale.setPaymentDescription(localeTexts.get("description"));
+                })
+        );
+
+        return ResponseEntity.ok(allowedPaymentMethods);
+    }
+
+    @GetMapping("/reservation/{reservationId}/get-selected-custom-payment-method-details")
```

Comment:
Same here. Path should be:

`/reservation/{reservationId}/selected-custom-payment-method-details`